src/Navbar.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Navbar.css';
+//import logo from 'logo-space'
+//import search image from 'search-image'
+
```

Comment:
Love those comments in the code, so we don't forget about importing the images as well !